### DISCUSSION TITLE
Best way to handle overflowing cell?

### DISCUSSION THREAD

**Discussion by zehawki** [table-library/react-table-library#45] [Q&A] [answered]
Is there a recommended way to handle text that's beyond the cell? Eg from X-grid - https://github.com/mui/mui-x/issues/417#issuecomment-954930443

**Comment by rwieruch** [comment, marked as answer]
The opt-in Theme applies the ... ellipsis for cells that are cut off. If the theme isn't used, the text just wraps vertically and the row grows in height. What would you want to see beyond that? After all, you have full control over the Cell, so you can put anything (e.g. a Tooltip) in there.

**Comment by proddy** [comment]
I couldn't get the ellipsis to show. It just always used the default clip. Perhaps somehow related to containers display property as discussed on https://stackoverflow.com/questions/17779293/css-text-overflow-ellipsis-not-working ?

my code:

```css
    BaseCell: `
      cursor: pointer;
      border-top: 1px solid transparent;
      border-right: 1px solid transparent;
      border-bottom: 1px solid transparent;
      padding-left: 16px;
      width: 124px;
      white-space: nowrap;
      overflow: hidden;
      text-overflow: ellipsis;
    `, etc....
```

**Reply by proddy** [reply]
fixed it, using the example code at https://react-table-library.com/?path=/docs/theming-themes--page

rtfm !
